sky/services/platform/interface.mojom

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module platform_interface;
```

Comment:
interface is redundant.  Everything in mojom is an interface.  I'd go with `module flutter.platform` in all these mojoms.